### ISSUE TITLE
AP[1:0] bits

### ISSUE THREAD

**Issue by jpark134** [sklaw/enee447project8_hw_template_Shuangqi_sessions#1]
In the example executions we can see in p8.pdf, I noticed when looking at the values of PTE[1] that the AP[1:0] bits were set to 10.  Just to double check, for us we should just leave these 2 bits as 00 correct?

**Comment by sklaw** [comment]
Yes since we [set all domain access mode to be "manager"](https://github.com/sklaw/enee447project8_hw_template_Shuangqi_sessions/blob/master/vm.c#L148-L149), which manes "Accesses are not checked against the permission bits in the translation tables" (B4-1558 in the arm doc).

**Comment by gogittossjx** [comment]
I have a follow up question. We know what the tex, b, and c bits are supposed to be, but what about all the other ones like domian, pxn, ns, etc? Should they all be 0

**Comment by sklaw** [comment]
Yes you can just set the other bits to be 0.

**Comment by gogittossjx** [comment]
And when we are doing the cache memory for tx, does it matter which inner and outer cache attribute we use or are all 4 good?

**Comment by sklaw** [comment]
> And when we are doing the cache memory for tx, does it matter which inner and outer cache attribute we use or are all 4 good?

Yes. All 4 should wok so you can just pick the one you see fit.
